core/feeders/newsletters.json

```
@@ -3,262 +3,314 @@
         {
             "name": "Golang Weekly",
             "link": "https://golangweekly.com/rss",
-            "web": "https://golangweekly.com"
+            "web": "https://golangweekly.com",
+			"category": "Programming"
```

```suggestion
			"category": "Golang"
```

We need to be specific here, for example changelog has a [podcast](https://changelog.com/gotime) specially for Go also.
In that case, we can include Go as a category and all the resources which are explicit to Go will share the same category.

We need to do this with all languages likewise JS, Ruby, Python etc